src/validators/delete-todo-validator.ts

```diff
@@ -0,0 +1,8 @@
+import { param, ValidationChain } from "express-validator";
+import { AppContext } from "@typings";
+
+const deleteTodoValidator = (): ValidationChain[] => [
+  param("id", "VALIDATION_ERRORS.INVALID_ID").isMongoId(),
```

Comment:
No need to add isMongoId check just add not empty and string as id check, because checking mongoId everywhere will increase our dependency over mongodb.